### ISSUE TITLE
[Feature Request] Multi-account switching

### ISSUE THREAD

**Issue by lazytownfan** [simple-login/Simple-Login-Android#118]
Currently only one SimpleLogin account may be logged in at any time.  There is currently no multi-account switching on the Android app.

### Explanation
The only way to switch between accounts (let's say two for the base case) on the Android app is to sign out of the first one and sign into the second account.  You must repeat the process to switch from the second account to return to the first account.  This is cumbersome if you want to switch between different SimpleLogin accounts on the Android app.

I have 2 SimpleLogin accounts, one for personal and the other for work.  Sometimes, I need to make new SimpleLogin aliases on the fly while I'm at work on my Android phone.  However, currently I only have my personal account on the Android app on my phone right now.  (For my current needs, solutions like a secondary profile on GrapheneOS for my work SimpleLogin account or installing SimpleLogin Android on a work profile are unnecessarily complex.)

I would say there could be a maximum of 5 accounts for the multi-account switching on the SimpleLogin app.

### Other note
I only think of this, as Proton Mail allows users to switch between accounts on Android (specifically up to 5 paid accounts, or 4 paid accounts and 1 free account).  I hope that if SimpleLogin implements multi-account switching on Android that SimpleLogin doesn't do something sneaky like what Proton Mail did.

**Comment by lazytownfan** [comment]
Yes, that's basically correct - I have 2 SimpleLogin accounts because I have 2 paid Proton accounts (one for personal, the other for work), so both basically come with 2 paid SimpleLogin accounts.

**Comment by ghost** [comment]
I use both Proton's simplelogin.io and the self-hosted SimpleLogin.

It would be very helpful if I could log in to different servers at the same time.
